db/osqlcomm.c

```diff
@@ -7143,7 +7162,7 @@ int osql_process_packet(struct ireq *iq, unsigned long long rqid, uuid_t uuid,
             NULL, NULL,                            /* vrec */
             NULL,                                  /*nulls, no need as no
                                                      ctag2stag is called */
-            *updCols, blobs, MAXBLOBS, &genid, dt.ins_keys, dt.del_keys,
+            *updCols, blobs, MAXBLOBS, &newgenid, dt.ins_keys, dt.del_keys,
```

Comment:
remove genid.

Reply:
genid is the vgenid of the function, newgenid will be written with the new genid if one is created.